Package.swift

```diff
@@ -128,10 +128,7 @@ let package = Package(
         ),
         .target(
             name: "SmithyRetries",
-            dependencies: [
-                "SmithyRetriesAPI",
-                .product(name: "AwsCommonRuntimeKit", package: "aws-crt-swift"),
-            ]
+            dependencies: ["SmithyRetriesAPI"]
```

Comment:
Removed the CRT dependency from SmithyRetries since the only CRT references in that target were unused and are now removed.